lib/plugins/aws/invokeLocal/resolvers/lambda.js

```diff
@@ -0,0 +1,23 @@
+const aws = require('aws-sdk');
+async function resolve(action, parameters, resourceName, resource, getCFResources, serverless, serviceConfig) {
+    switch(action) {
+        case 'Ref':
+            return resource.Properties.FunctionName;
+        case 'Fn::GetAtt':
+            switch(parameters[1]) {
+                case 'Arn':
+                    const lambda = new aws.Lambda(serviceConfig);
+                    const func = await lambda.getFunction({
+                        FunctionName: resource.Properties.FunctionName
+                    }).promise();
+                    if(!func) {
+                        return action;
+                    }
+                    return func.Configuration.FunctionArn;
+            }
```

Comment:
We should crash on any non `Arn` parameter.

Reply:
I'm wondering if we should crash, or just return the current value and let the deployment crash.  Crashing debugging on a get attribute parameter seems like it would cause issues if GetAtt is extended in cloudformation.  Not setting the value at least lets the user debug, but might not set the value correctly.  Not exactly sure how to solve this one... maybe log that the attribute name is not supported?

Reply:
I would meaningfully crash, as otherwise we're clearly be providing an incomplete environment for local invocation